packages/rspack/package.json

```diff
@@ -66,7 +66,7 @@
     "tsx": "^4.19.3",
     "typescript": "^5.7.3",
     "watchpack": "^2.4.2",
-    "webpack-dev-server": "5.0.4",
+    "webpack-dev-server": "5.2.0",
```

Comment:
@LingyuCoder Can we bump webpack-dev-server now? I remember that there were some reasons that forced us to lock the version.